storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -264,7 +269,9 @@ func (c *PrometheusConverter) FromMetrics(ctx context.Context, md pmetric.Metric
 				}
 			}
 		}
-		addResourceTargetInfo(resource, settings, mostRecentTimestamp, c)
+		if earliestTimestamp < pcommon.Timestamp(math.MaxUint64) && latestTimestamp > 0 {
```

Comment:
Nit: Is it valid to have samples with ts = 0? If so, we could get in a case where a valid sample with ts 0 is written but doesn't have a target_info. In real scenarios, having ts = 0 is probably unlikely though

Reply:
I dropped the check for `latestTimestamp > 0` @fionaliao, since I think it's overkill as explained above. Thanks!